docs/_static/jdaviz.css

```diff
@@ -15,12 +16,6 @@
   margin: 0px;
 }
 
-.sd-card .sd-card-img-top {
-  height: 52px;
-  width: 52px;
-  margin-left: auto;
-  margin-right: auto;
-}
```

Comment:
looks like `:img-top` is no longer used anywhere ✅

Reply:
Yes